<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>系统功能</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>学生信息管理</string>
     </property>
     <item>
      <property name="text">
       <string>添加</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>修改</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>删除</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>学生排序</string>
     </property>
     <item>
      <property name="text">
       <string>按姓名排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按学号排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按宿舍排序</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>查找学生</string>
     </property>
     <item>
      <property name="text">
       <string>按姓名查找</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按学号查找</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按宿舍查找</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>保存</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>加载</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>51</y>
      <width>441</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加载</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>学生信息管理</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>学生排序</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查找学生</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>加载</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>修改 </string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>姓名</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>学号</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>宿舍</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>姓名</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>学号</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>宿舍</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
